src/main/java/de/flux/playground/deckcompare/dto/Deck.java

```diff
@@ -15,10 +16,23 @@
 @XmlAccessorType(XmlAccessType.FIELD)
 public class Deck {
 
+    private static final int DEFAULT_SECTIONS = 2;
+    private static final int CARDS_SECTION = 1;
+
     @XmlAttribute(name = "game")
     private String id;
     @XmlElement(name = "section")
     private List<Section> sections;
     private String notes;
 
+    public List<Card> getCards() {
+        List<Card> cards = new ArrayList<Card>();
```

Comment:
Would be better to assign `Collections.emptyList()` here. Otherwise this method will create a new `ArrayList`each time it is called.